cypress/support/add-place.js

```diff
@@ -0,0 +1,17 @@
+import { leftNavbar, sequelize } from './cssCommonSelectors';
+import components from './components';
+
+function addPlace(name, description){
+  cy.loginSuccess()
+    .get(leftNavbar.sequelize.favouritePlaces).click()
+    .get(components.common.actionButton).contains('Create new').click()
+    .get(sequelize.inputs.name).type(name)
+    .get(sequelize.inputs.description).type(description)
+    .get('[for="userId"]').next().click()
```

Comment:
Can it be extracted to variable/constant?

Reply:
Agree. Fixing